xml/transactional_updates.xml

```diff
@@ -0,0 +1,444 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook" version="5.0"
+ xmlns:xi="http://www.w3.org/2001/XInclude"
+ xmlns:xlink="http://www.w3.org/1999/xlink"
+ xml:id="cha.transactional-updates" xml:lang="en">
+ <title>Transactional Updates</title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+  <abstract>
+   <para>
+    Transactional updates are available in &productname; as a technology
+    preview, for updating &slsa; when the root filesystem is read-only. 
+    Transactional updates are atomic (all updates are applied only if all 
+    updates succeed) and support rollbacks. It does not affect a running system 
+    as no changes are applied until after the system is rebooted.
+   </para>
+
+   <para>
+    Transactional updates are run daily by the
+    <command>transactional-update</command> script. The script checks for
+    available updates. If there are any updates, it creates a new snapshot of
+    the root file system in the background, and then fetches updates from the
+    release channels. After the new snapshot is completely updated, it is
+    marked as active and will be the new default after the next reboot of the
+    system. When <command>transactional-update</command> is set to run
+    automatically (which is the default behavior) it also reboots the system.
+    Both the time that the update runs and the reboot maintenance window are 
+    configurable.
+   </para>
+
+   <para>
+    Only packages that are part of the snapshot of the root file system can be
+    updated. If packages contain files that are not part of the snapshot, the
+    update could fail or break the system.
+   </para>
+
+   <para>
+    RPMs that require a license to be accepted cannot be updated.
+   </para>
+  </abstract>
+ </info>
+ 
+ <sect1 xml:id="sec.tu-limitations">
+  <title>Limitations of Technical Preview</title>
+
+  <para>
+   As a technical preview, there are certain limitations in functionality. The
+   following packages will not work with <command>transactional-update</command>. 
+   In the <filename>/srv</filename> directory:
```

Comment:
I would remove the /srv and /var directory separation and move the two sections together. This was for developers, that they know where the problem with their package is, but will only confuse customers.